packages/kmod-5.10-nvidia/nvidia-tesla-tmpfiles.conf.in

```diff
@@ -0,0 +1,4 @@
+C /etc/drivers/driver-config.toml - - - - /usr/share/factory/nvidia/tesla/NVIDIA_VERSION/build-config.toml
```

Comment:
This would be somewhat more ergonomic if `driverdog` read all the config files in the directory - then the tmpfile could be in `/usr/share/factory` under its final name (`nvidia-tesla-VERSION.toml`).